README.md

```diff
@@ -52,7 +52,7 @@ Published releases are available on maven central.
     <dependency>
       <groupId>io.opentelemetry</groupId>
       <artifactId>opentelemetry-api</artifactId>
-      <version>0.6.0-SNAPSHOT</version>
+      <version>0.7.0</version>
```

Comment:
Was this intentional? I thought master should always have a -SNAPSHOT version and releases happen only from release/ branches?